music/src/core/player.ts

```diff
@@ -139,9 +139,18 @@ export abstract class BasePlayer {
    * specified, will use either the qpm specified in the sequence or the
    * default of 120. Only valid for quantized sequences.
    * @returns a Promise that resolves when playback is complete.
+   * @throws {Error} If this or a different player is currently playing.
    */
 
   start(seq: INoteSequence, qpm?: number): Promise<void> {
+    if (this.isPlaying()) {
+      throw new Error(`Cannot start playback while "${this.getPlayState()}".`);
```

Comment:
I think you can just say "Cannot start playback; player is already playing" to be more clear.